swiftui/ChartboostMediationDemo/Ads/AdTypeSelectionView.swift

```diff
@@ -28,10 +29,14 @@ struct AdTypeSelectionView: View {
                             .padding(.vertical, 8)
                         Spacer()
                     }
-
+                    HStack {
+                        Toggle("Use Fullscreen API", isOn: $useFullscreenApi)
+                            .padding(.horizontal)
+                            .tint(.accentColor)
+                    }
```

Comment:
nit: is it necessary to embed the Toggle inside a HStack? It's only one view so it shouldn't make a difference